src/js/view/actions/dispatchActions.js

```diff
@@ -74,12 +74,20 @@ export const updateLastSeenLogSizes = (dispatch, path, fileSize) => {
   });
 };
 
-export const setFileData = (dispatch, filePath, fileSize, history) => {
+export const setFileData = (
+  dispatch,
+  filePath,
+  fileSize,
+  history,
+  metaData
```

Comment:
metaData feels a little vague. We are not really using it for now since we did not need it for the MVP at the moment. It will probably be useful though to keep for the coming issues. How about changing it to `logLinesMetaData { startingByteOfLines: { [sourcePath]: 12345  } }` or something to the like?